game.rb

```diff
@@ -0,0 +1,128 @@
+require_relative 'dictionary'
+
+class Game
+  attr_reader :word, :guess
+
+  def initialize(word = nil)
+    @dictionary = Dictionary.new.trim_dictionary
+    @word = word || @dictionary.sample(1)[0].split('')
```

Comment:
Why not just `@dictionary.sample` ? Am I missing something?

Reply:
Nope, @dictionary.sample works perfectly